lib/cisco_node_utils/cmd_ref/fabricpath.yaml

```diff
@@ -69,17 +67,23 @@ loadbalance_algorithm:
   get_command: "show run fabricpath all"
   get_value: '/^fabricpath load\-balance (\S+)\s*$/'
   set_value: "<state> fabricpath load-balance <algo>"
-  N5k:
+  N5k: &loadbal_alg_n5_n6
```

Comment:
n5k and n6k are implied by the yaml entries themselves.  I would change this to be just `&loadbal_alg` and `*loadbal_alg` or something else meaningful minus the platforms.